pkg/apic/apiservicerevision.go

```diff
@@ -101,25 +118,53 @@ func (c *ServiceClient) processRevision(serviceBody *ServiceBody) error {
 	return nil
 }
 
-func (c *ServiceClient) getRevisionCount(queryString string) int {
+func (c *ServiceClient) getRevisions(queryString string) ([]*management.APIServiceRevision, int, error) {
+	apiServiceRevisions := make([]*management.APIServiceRevision, 0)
+
 	queryParams := map[string]string{
 		"query":    queryString,
-		"fields":   "id",
+		"fields":   "metadata.id,name,tags",
```

Comment:
I guess we do not necessarily need the id now, since we are getting the other items

Reply:
we can... updating fields